src/Components/CreateImageWizard/formComponents/ReviewStepTextLists.js

```diff
@@ -558,6 +558,15 @@ export const ImageDetailsList = () => {
             <TextListItem component={TextListItemVariants.dd}>
               {getState()?.values?.['image-name']}
             </TextListItem>
+            <TextListItem
```

Comment:
The description should be rendered conditionally so it doesn't show in the review when empty. Because of `{getState()?.values?.['image-name'] && (` it renders when the imageName is filled, so we'll just need to split the condition for name and description.

![Screenshot from 2023-07-25 12-40-49](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/13834f38-9e88-4a81-89ce-d7a60e8eec78)

Reply:
done, thank you